api4/job.go

```diff
@@ -45,17 +45,8 @@ func getJob(c *Context, w http.ResponseWriter, r *http.Request) {
 
 func downloadJob(c *Context, w http.ResponseWriter, r *http.Request) {
 	config := c.App.Config()
-	const FILE_PATH = "export/%s/%s"
-	fileInfo := map[string]map[string]string{
-		"csv": {
-			"fileName": "csv_export.zip",
-			"fileMime": "application/zip",
-		},
-		"actiance": {
-			"fileName": "actiance_export.zip",
-			"fileMime": "application/zip",
-		},
-	}
+	const FILE_PATH = "export/%s"
```

Comment:
This path should probably be constructed with something like [`filepath.Join`](https://golang.org/pkg/path/filepath/#Join) so that it works with our Windows builds.